hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/acl/OMBucketAddAclRequest.java

```diff
@@ -134,7 +127,9 @@ void onComplete(boolean operationResult, IOException exception,
         exception, getOmRequest().getUserInfo()));
 
     if (operationResult) {
-      LOG.debug("Add acl: {} to path: {} success!", getAcls(), getPath());
+      if (LOG.isDebugEnabled()) {
```

Comment:
Is this for performance? Maybe there is no problem here.
It is recommended to use parameterized messages.
Please see http://www.slf4j.org/faq.html#logging_performance

Reply:
Thanks for review! I saw the usage in OMBucketSetAclRequest. It is introduced by HDDS-2266.
There are many class related to the patch, may be we can fix in separate JIRAs?

Reply:
You can file a new jira to track HDDS-2266, but it's unnecessary change here.

Reply:
OK, I will rollback it.